### PR TITLE
Nicks branch

### PR DESCRIPTION
**Pull request by FavourAmayo**
Added conditional rendering of LoginPage form

### CODE REVIEW

**Review by PowerOfKaishin — APPROVED** [review]

**Comment by theresa-li** [comment]
Is the merge to master branch intentional, or did you mean to merge into dev?

File: src/Components/App.js [hunk 1]
```diff
@@ -6,13 +6,11 @@ import { useSelector } from "react-redux";
 //Page Components
 import MyBatches from "./batches/MyBatches";
 import LoginPage from "./login/LoginPage";
-import LogoutPage from "./login/LogoutPage";
```
Comment (theresa-li) [inline src/Components/App.js:9]: Is there a reason for removing the LogoutPage?

File: src/Components/App.js [hunk 2]
```diff
@@ -28,17 +26,14 @@ import { Forbbiden, NotFound } from "./Common/ErrorPages";
...
-      {token ? <Drawer /> : null}
+      {/* {token ? <Drawer /> : null} */}
+      {isLoggedIn ? <Drawer /> : null}
       <Switch>
-        {/* conditional render the home page if they're logged in or login page if not */}
-        {token ? (
-          <Route path="/Home" component={HomePage} exact />
-        ) : (
-          <Route path="/login" component={LoginPage} exact />
-        )}
```
Comment (theresa-li) [inline src/Components/App.js:41]: Is there a reason for removing the conditional rendering?

File: src/Components/Common/HomePage.js [hunk 3]
```diff
@@ -1,10 +0,0 @@
-import React from "react";
-
-export default function HomePage() {
-  return (
-    <div>
-      <h1>Welcome To Revature's Client Engagement Portal</h1>
-      <h2>Please select from the options on the left</h2>
-    </div>
-  );
-}
```
Comment (theresa-li) [inline src/Components/Common/HomePage.js:10]: Why was this file deleted?

File: src/Components/App.js (hunk 1 above)
Comment (FavourAmayo) [reply src/Components/App.js:9]: We don't need LogoutPage. When logging out, the user should go back to the homepage

File: src/Components/login/LogoutPage.js [hunk 4]
```diff
@@ -1,11 +0,0 @@
...
-
-export default function LogoutPage() {
-  return (
-    <div>
-      <h1>Thank You For Using Revature's Services</h1>
-      <h2>Please select Login on the left</h2>
-    </div>
-    //TODO: make it clear user in redux and redirect to Login Page
-  );
-}
```
Comment (theresa-li) [inline src/Components/login/LogoutPage.js:11]: Why was the LogOutPage file removed?

**Review by theresa-li — COMMENTED** [review]
It looks like some files are missing. Is this intentional?

File: src/Components/App.js (hunk 1 above)
Comment (theresa-li) [reply src/Components/App.js:9]: That sounds good to me.

**Review by theresa-li — APPROVED** [review]
